includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -181,11 +177,10 @@ private function getLangWithFallback( $lang ) {
 			'user-register' => 'https://www.wikia.com/register?uselang=ja',
 		],
 		'pl' => [
-			'explore-wikis' => 'http://fandom.wikia.com/explore?uselang=pl',
-			'fandom-logo' => 'http://pl.wikia.com',
+			'explore-wikis' => 'http://pl.wikia.com/Wikia',
```

Comment:
It's the same redirect issue which I wrote about here https://github.com/Wikia/app/pull/11496/files#r81123582. It would be the same link without `/Wikia` for now.